libs/@guardian/source-react-components-development-kitchen/src/footer-with-contents/FooterLinks.tsx

```diff
@@ -19,7 +20,7 @@ type FooterLink = AnchorHTMLAttributes<HTMLAnchorElement> & {
 	/**
 	 * The URL of the link
 	 */
-	href: string;
+	href?: string;
```

Comment:
Does this need to be optional if the FooterLink and FooterButton are two distinct types wouldn't href always be present in a FooterLink?

Reply:
@rBangay see comment above https://github.com/guardian/csnx/pull/940/files#r1400352844